src/main/java/it/gov/pagopa/pu/debtpositions/exception/custom/InvalidInstallmentStatusException.java

```diff
@@ -0,0 +1,8 @@
+package it.gov.pagopa.pu.debtpositions.exception.custom;
+
+public class InvalidInstallmentStatusException extends RuntimeException {
```

Comment:
Can we generalize this exception? the message should info about the entity, the status from and status to
```suggestion
public class InvalidStatusTransitionException extends RuntimeException {
```